.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,65 @@
+name: tag-release
+on:
+  push:
+    tags:
+      - "*.*.*"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: always()
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Set output
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+      - name: Check output
+        env:
+          RELEASE_VERSION: ${{ steps.vars.outputs.tag }}
+        run: |
+          echo $RELEASE_VERSION
+          echo ${{ steps.vars.outputs.tag }}
+      - name: Init environment
+        run: cd docker && ./init.sh && cat .env && cd ../
```

Comment:
Is this step necessary?

Reply:
Good point! That was not mandatory. Also, I removed the 'List items' step. That was also a debug left-over